Engine/Component/Component.cpp

```diff
@@ -17,6 +17,10 @@ void Component::Load(const Config &config)
 	SpecializedLoad(config);
 }
 
+void Component::CloneBase(Component* component) const
+{
+	*component = *this;
+}
```

Comment:
for what is used now this Clone Base function? What happened or what's the difference with the other Clone?

Reply:
I need to clone also the base component parameters and for not oding just *component = *this; everywhere I created the function.